Formula/rswift.rb

```diff
@@ -1,14 +1,12 @@
 class Rswift < Formula
   desc "Get strong typed, autocompleted resources like images, fonts and segues"
   homepage "https://github.com/mac-cain13/R.swift"
-  url "https://github.com/mac-cain13/R.swift.git",
-      :tag      => "v5.2.0",
-      :revision => "ec748b51d016e42016c93f8df8acb5e8f3635ccc"
+  url "https://github.com/mac-cain13/R.swift/archive/v5.2.1.tar.gz"
```

Comment:
When 5.2.2 is out, we could update this to `url "https://github.com/mac-cain13/R.swift/releases/download/v5.2.2/release.tar.gz"`

Reply:
Thanks!